backend/local.env

```diff
@@ -18,4 +18,4 @@ JWT_REFRESH_TOKEN_EXPIRATION_TIME=31540000
 # small values used for testing tokens
 # JWT_REFRESH_TOKEN_EXPIRATION_TIME=10
 
-ACTIVE_PROFILES=test_data,test,e2e
+ACTIVE_PROFILES=test_data,e2e
```

Comment:
this is fine but for now, if we want to run tests locally, we would need to add this back manually. i will find a way so that we don't have to do this and that the tests have separate env file